api/src/main/java/com/epam/pipeline/manager/datastorage/StorageProviderManager.java

```diff
@@ -94,6 +95,10 @@ public void restoreFileVersion(AbstractDataStorage dataStorage, String path, Str
             throws DataStorageException {
         getStorageProvider(dataStorage).restoreFileVersion(dataStorage, path, version);
     }
```

Comment:
Do not forget to place empty line between methods

Reply:
fixed